popUp.js

```diff
@@ -0,0 +1,412 @@
+    techskills: ['HTML', 'Ruby on Rails', 'CSS', 'JavaScript'],
+    image: 'src="Images_Background/Snapshoot Portfolio 4_Art Printing.svg"',
+    liveButton: '#',
+    sourceButton: '#',
+  },
+  desktop3: {
+    name: 'Facebook 360',
+    role: ['FACEBOOK', 'Full Stack Dev', '2015'],
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing and typesetting industry.<br><br>Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s.",
+    techskills: ['HTML', 'Ruby on Rails', 'CSS', 'JavaScript'],
+    image: 'src="Images_Background/Snapshoot Portfolio 1_Orange.svg"',
+    liveButton: '#',
+    sourceButton: '#',
+  },
+  desktop4: {
+    name: 'Multi-Post Stories',
+    role: ['CANOPY', 'Back End Dev', '2015'],
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing and typesetting industry.<br><br>Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s.",
+    techskills: ['HTML', 'CSS', 'JavaScript'],
+    image: 'src="Images_Background/Snapshoot Portfolio 2_Availability.svg"',
+    liveButton: '#',
+    sourceButton: '#',
+  },
+};
+
+const popUpContainer = document.getElementById('mobilepopup-container');
+const desktopPopUpContainer = document.getElementById('desktoppopup-container');
+const buttons = document.querySelectorAll('.get-button');
+
+if (window.innerWidth <= 768) {
```

Comment:
- [ ] The mobile version of the design is broken, kindly make sure the design layout displays properly when you access it from mobile

|Mobile|Web view|Figma design|
|-|-|-|
|![Screenshot_20230518_045717](https://github.com/HFG43/Portfolio_Mobile_First/assets/25227022/dfe986a5-84ab-43f8-8375-bfe3cdf88cfa)|![Screenshot_20230518_045733](https://github.com/HFG43/Portfolio_Mobile_First/assets/25227022/3dcf2aaf-c739-4bfc-a48c-64a5998c70ca)|![Screenshot_20230518_050718](https://github.com/HFG43/Portfolio_Mobile_First/assets/25227022/6fefba28-18a5-4737-b6ff-450637923705)|